config/swagger.yaml

```diff
@@ -201,7 +148,7 @@ paths:
       parameters:
         - in: body
           name: body
-          description: Put a new transaction to node
+          description: a new transaction
```

Comment:
A small typo: "A new ..."

Reply:
There is no verb in this description and hence it is not a sentence, therefore I was not using a capital A here.

Reply:
Still, it would be consistent with "A new block" above and "A spend transaction" below to use a capital A.